i18n/appointments/locale_en.json

```diff
@@ -177,4 +177,72 @@
   "APPOINTMENT_ACCEPT_CONFIRM_MESSAGE": "Are you sure, you want to accept the appointment invite?",
   "JOIN_TELE_CONSULTATION":"Join Teleconsultation",
   "COPY_LINK": "Copy TeleConsultation Link"
```

Comment:
, missing

Reply:
nice catch @buvaneswari-arun Will fix that and push it to the PR